app/controllers/api/v1/protected_application_controller.rb

```diff
@@ -1,7 +1,7 @@
 module Api
   module V1
     class ProtectedApplicationController < ApplicationController
-      before_action :authenticate_request
+      before_action :authenticate_request, only: %i[index create destroy]
```

Comment:
I don't think this is ideal. With this change, you are unprotecting all actions on all controllers that inherit from `ProtectedApplicationController`. Is that what you want? Usually, we build these things with a `secure by default` philosophy, I think this goes against that idea

Reply:
yeah i agree